.github/workflows/release-sdk.yml

```
@@ -463,5 +463,6 @@ jobs:
             {
               "version": "${{ inputs.version }}",
               "actor": "${{ github.actor }}",
-              "actor_id": "${{ github.actor_id }}"
+              "actor_id": "${{ github.actor_id }}",
+              "mode": ${{ inputs.mode }}
```

```suggestion
              "repository": "${{ github.repository }}",
              "workflow_ref": "${{ github.workflow_ref }}",
              "run_id": "${{ github.run_id }}",
              "mode": "${{ inputs.mode }}"
```

The value from `mode` needs to be quoted (it's a string, not a boolean). This brings in the rest of my change from https://github.com/wandb/wandb/pull/11009.